src/Parameter.cpp

```diff
@@ -31,6 +32,7 @@ struct ParameterContents {
         // dense vectorization. You can unset it by setting it to a
         // null expression. (param.set_stride(0, Expr());)
         stride_constraint[0] = 1;
+        host_alignment = 1;
```

Comment:
out of curiosity: is there any modern system for which we'll see an allocator return values that aren't least 2 (or 4) byte aligned?